include/motor-control/core/stepper_motor/motor_interrupt_handler.hpp

```diff
@@ -422,6 +431,11 @@ class MotorInterruptHandler {
             can::messages::ErrorMessage{.message_index = message_index,
                                         .severity = severity,
                                         .error_code = err_code});
+        if (err_code == can::ids::ErrorCode::hardware) {
```

Comment:
I think we should make an error code that's specifically "motor driver error detected" if we're going to key "query the motor driver status" off it.